checker-qual/src/main/java/org/checkerframework/framework/qual/InheritedAnnotation.java

```diff
@@ -8,7 +8,13 @@
 
 /**
  * A meta-annotation that specifies if a declaration annotation should be inherited. This should not
- * be written on type annotations.
+ * be written on type annotations. Unlike {@link java.lang.annotation.Inherited}, this
+ * meta-annotation causes the declaration annotation to be inherited even if it is used to annotate
+ * something other than a class.
+ *
+ * <p>The Checker Framework does respect {@link java.lang.annotation.Inherited} and will only
```

Comment:
Is this "does" or "does not"?

Reply:
does.

Reply:
I'm confused by the wording.
 * "`@Inherited` is respected" implies that meta-annotating with `@Inherited` is enough to inherit the annotation, on a class.
 * "will only inherit declaration annotations on a class if .. meta-annotated with `@InheritedAnnotation`" implies that `@InheritedAnnotation` is needed to cause inheritance.

This is in conflict.  Which is right?

Reply:
The first one is correct.  I rewrote the comment to make it clearer.